docs/contributors/getting-started.md

```diff
@@ -125,6 +125,20 @@ To use Prettier, you should install the [Prettier - Code formatter](https://mark
 
 This will use the `.prettierrc.js` file in the root folder of the Blocks plugin repository and the version of Prettier that is installed in the root `node_modules` folder.
 
+## Testing
+
+You’ll find a handful of scripts in `package.json` that performs the automated tests and linting. You can run the following commands to execute automated tests in your terminal:
+
+- JS tests: `npm run test`
+- End-to-End `tests:npm run test:e2e` 
+- PHP tests: `npm run test:php` 
+
+You can also use [Docker](https://www.docker.com/) to run end-to-end or php tests.
```

Comment:
I don't think this should be here, the npm scripts for PHP and E2E run in Docker anyway, maybe we should just skip this line, keep the line below about `npm run wp-env` though 👍🏼